bundles/core/org.eclipse.smarthome.core.test/META-INF/MANIFEST.MF

```diff
@@ -19,12 +19,10 @@ Import-Package:
  org.eclipse.smarthome.test.java,
  org.hamcrest;core=split,
  org.junit,
+ org.junit.rules;version="4.7.0",

```

Comment:
A lot of other org.junit.rules imports are versioned with "4.0.0". Is this here 4.7.0 by accident?

Reply:
No, this is by intention: see https://junit.org/junit4/javadoc/4.12/org/junit/rules/TestName.html (it was added with 4.7)